### PR TITLE
Tox + travis

### PR DESCRIPTION
**Pull request by jlward**

### CODE REVIEW

File: tagging/tests/__init__.py [hunk 1]
```diff
@@ -0,0 +1,124 @@
+tags = [
```
Comment (jlward) [inline tagging/tests/__init__.py:1]: This was moved.

File: tagging/tests/settings.py [hunk 2]
```diff
@@ -3,21 +3,12 @@
...
-DATABASE_HOST = os.environ.get("TAGGING_DATABASE_HOST", "localhost")
-
-if test_engine == "sqlite":
-    DATABASE_NAME = os.path.join(DIRNAME, 'tagging_test.db')
-    DATABASE_HOST = ""
-elif test_engine == "mysql":
-    DATABASE_PORT = os.environ.get("TAGGING_DATABASE_PORT", 3306)
-elif test_engine == "postgresql_psycopg2":
-    DATABASE_PORT = os.environ.get("TAGGING_DATABASE_PORT", 5432)
+DATABASES = {
```
Comment (jlward) [inline tagging/tests/settings.py:6]: Update DB stuff.

File: tagging/tests/tags.txt [hunk 3]
```diff
@@ -1,122 +0,0 @@
-NewMedia 53
```
Comment (jlward) [inline tagging/tests/tags.txt:1]: Removed from txt file. Moved to `__init__` file.

File: tagging/tests/tests.py [hunk 4]
```diff
@@ -9,6 +8,7 @@
 from tagging.models import Tag, TaggedItem
 from tagging.tests.models import Article, Link, Perch, Parrot, FormTest, FormTestNull
 from tagging.utils import calculate_cloud, edit_string_for_tags, get_tag_list, get_tag, parse_tag_input
+from tagging.tests import tags as default_tags
```
Comment (jlward) [inline tagging/tests/tests.py:11]: This is almost all whitespace.

File: tagging/tests/tests.py [hunk 5]
```diff
@@ -158,49 +158,49 @@ def test_with_invalid_input(self):
...
-    
+
     def test_nonexistent_tag(self):
         self.assertEquals(get_tag('mouse'), None)
 
 class TestCalculateCloud(TestCase):
     def setUp(self):
         self.tags = []
-        for line in open(os.path.join(os.path.dirname(__file__), 'tags.txt')).readlines():
+        for line in default_tags:
```
Comment (jlward) [inline tagging/tests/tests.py:174]: This is updated.

File: tagging/tests/settings.py (hunk 2 above)
Comment (winhamwr) [inline tagging/tests/settings.py:6]: Any reason you dropped the postgres and mysql tests?

**Comment by winhamwr** [comment]
This looks good, but are you planning on forking it to the PolicyStat org?

File: tagging/tests/settings.py (hunk 2 above)
Comment (jlward) [inline tagging/tests/settings.py:6]: Because we are the only people using this lib.

**Comment by jlward** [comment]
> This looks good, but are you planning on forking it to the PolicyStat org?

When I get all done with the work that needs to be done, I can transfer ownership to PolicyStat.

**Pull request merged**